arbnode/transaction_streamer.go

```diff
@@ -580,14 +580,15 @@ func (s *TransactionStreamer) countDuplicateMessages(
 	return curMsg, false, nil, nil
 }
 
-func (s *TransactionStreamer) logReorg(pos arbutil.MessageIndex, dbMsg *arbstate.MessageWithMetadata, newMsg *arbstate.MessageWithMetadata, force bool) {
+func (s *TransactionStreamer) logReorg(pos arbutil.MessageIndex, dbMsg *arbstate.MessageWithMetadata, newMsg *arbstate.MessageWithMetadata, confirmed bool) {
 	sendLog := force
```

Comment:
This commit isn't compiling FYI, I think you just missed this in the argument rename :)